atomate/qchem/fireworks/tests/test_core.py

```diff
@@ -0,0 +1,191 @@
+# coding: utf-8
+
+from __future__ import division, print_function, unicode_literals, absolute_import
+
+import os
+import unittest
+
+from atomate.qchem.firetasks.write_inputs import WriteInputFromIOSet
+from atomate.qchem.firetasks.run_calc import RunQChemCustodian
+from atomate.qchem.firetasks.parse_outputs import QChemToDb
+from atomate.qchem.fireworks.core import *
```

Comment:
avoid import * like this

Better if you can use an IDE to help you add imports automatically. See also:

https://stackoverflow.com/questions/2360724/what-exactly-does-import-import?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

Reply:
Thanks, fixed